Kernel/Devices/USB/UHCIController.cpp

```diff
@@ -350,11 +358,11 @@ void UHCIController::do_debug_transfer()
     // Let's begin....
     setup_td->set_status(0x18800000);
     setup_td->set_token(0x00E0002D);
-    setup_td->set_buffer_address(m_td_buffer_region->physical_page(0)->paddr().get());
+    setup_td->set_buffer_address(td_buffer_region->physical_page(0)->paddr().get());
 
     data_td->set_status(0x18800000);
     data_td->set_token(0x00E80069);
-    data_td->set_buffer_address(m_td_buffer_region->physical_page(0)->paddr().get() + 16);
+    data_td->set_buffer_address(td_buffer_region->physical_page(0)->paddr().get() + 16);
 
     response_td->set_status(0x19800000);
     response_td->set_token(0xFFE800E1);
```

Comment:
???

Reply:
This entire function should probably be removed to be honest.